### DISCUSSION TITLE
Wind speed in m/s

### DISCUSSION THREAD

**Discussion by ICDeadPpl** [rocksdanister/weather#38] [Ideas]
I'd like to see the wind speed in m/s, as this is the way it's reported in weather reports in Sweden.

**Comment by rocksdanister** [comment]
Is this about Lively Wallpaper app? or Lively Weather?
https://www.rocksdanister.com/weather
https://www.rocksdanister.com/lively

**Reply by ICDeadPpl** [reply]
Lively Weather
